packages/cli/src/Db.ts

```diff
@@ -179,6 +180,8 @@ export async function init(
 	collections.InstalledPackages = linkRepository(entities.InstalledPackages);
 	collections.InstalledNodes = linkRepository(entities.InstalledNodes);
 	collections.WorkflowStatistics = linkRepository(entities.WorkflowStatistics);
+	collections.FeatureConfig = linkRepository(entities.FeatureConfig);
```

Comment:
Instead of the feature config entity, can't we use the existing `settings` table? It's basically the same thing and as long as we set `loadOnStartup` to false, it's all right.

Reply:
No reason not to do it like that @netroy @krynble